keps/993-two-phase-admission/README.md

```diff
@@ -163,22 +163,20 @@ type AdmissionCheckSpec struct {
 	// A reference to the additional parameters for the check. 
 	Parameters *AdmissionCheckParametersReference `json:"parameters,omitempty"`
 
-	// How should the workload book quota (and possibly preempt others) 
-	// before the check pass. The default is NoBooking.
-	// If different modes are used for different AdmissionChecks,
-	// the most restrictive one is used.
-	QuotaBooking *QuotaBookingMode `json:"quotaBooking,omitempty"`
+	// preemptionPolicy determines when to issue preemptions for the Workload,
+	// if necessary, in relationship to the status of the admission check.
+	// The possible values are:
+	// - `Anytime`: No need to wait for this check to pass before issuing preemptions.
+	//   Preemptions might be blocked on the preemptionPolicy of other AdmissionChecks.
+	// - `AfterCheckPassedOrOnDemand`: Wait for this check to pass pass before issuing preemptions,
+	//   unless this or other checks requests preemptions through the Workload's admissionChecks.
+	// Defaults to `Anytime`.
+	PreemptionPolicy *PreemptionPolicy `json:"preemptionPolicy,omitempty"`
 }
 
-// Definitons of QuotaBookingModes.
-// BookAfterQuotaCheck is more restrictive than NoBooking.
 const (
-	// No booking, unless a check explicitely asks for it reporting quota exhausted.
-	NoBooking QuotaBookingMode = "NoBooking"
-	
-	// Book quota (and preempt) immediately after the basic quota checks 
-	// are done, without waiting for the additional checks to cpmplete.
-	BookAfterQuotaCheck QuotaBookingMode = "BookAfterQuotaCheck"
+	NoBooking PreemptionPolicy = "Anytime"
```

Comment:
Match name with the value. Here and below.

Reply:
Done